src/screens/HtmlScreen.js

```diff
@@ -22,7 +29,8 @@ export const HtmlScreen = ({ navigation, route }) => {
   const [refreshing, setRefreshing] = useState(false);
   const trackScreenViewAsync = useTrackScreenViewAsync();
 
-  if (!query || !queryVariables?.name) return null;
+  if (!query || !queryVariables?.name)
+    return <EmptyMessage title="Im Moment konnte kein anzuzeigender Inhalt gefunden werden." />;
```

Comment:
we try to avoid hardcoded texts and use references to texts from the file `src/config/texts`.
i could think of something like `texts.empty.content` here.
the text itself could be changes to: "Schade, es wurde kein passender Inhalt gefunden, bitte zu einem späteren Zeitpunkt erneut versuchen."

Reply:
I will update the code in this way.

Reply:
you have missed this one, right? it should be a different text here as for the lists.

Reply:
Yes true. Sorry, I thought it was the same text. I'm updating right now.